I cannot get this to build on my linux machine:
```
>    └─ zig build-exe ghostty Debug native-native 1 errors
┃        > src/renderer/OpenGL.zig:2347:63: error: expected type 'f32', found 'f64'
┃        >         @as(f32, @floatFromInt(self.draw_background.r)) / 255 * self.config.background_opacit…
┃        >         ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
┃        > pkg/opengl/draw.zig:5:22: note: parameter type declared here
┃        > pub fn clearColor(r: f32, g: f32, b: f32, a: f32) void {
┃        >                      ^~~
┃        > referenced by:
┃        >     drawFrame: src/renderer/OpenGL.zig:2289:13
┃        >     render: src/apprt/gtk/Surface.zig:669:35
┃        >     remaining reference traces hidden; use '-freference-trace' to see all reference traces
┃        > error: the following command failed with 1 compilation errors:
┃        > /nix/store/v9xybfsc2iyfd602s192pymyrrncni0n-zig-0.13.0/bin/zig build-exe /build/source/.zig-c…
┃        > Build Summary: 94/97 steps succeeded; 1 failed (disable with --summary none)
┃        > install transitive failure
┃        > └─ install ghostty transitive failure
┃        >    └─ zig build-exe ghostty Debug native-native 1 errors
```
Appears to be a type mismatch issue.

--- a/src/renderer/Metal.zig
+++ b/src/renderer/Metal.zig
@@ -1186,9 +1186,9 @@ pub fn drawFrame(self: *Metal, surface: *apprt.Surface) !void {
                 attachment.setProperty("storeAction", @intFromEnum(mtl.MTLStoreAction.store));
                 attachment.setProperty("texture", screen_texture.value);
                 attachment.setProperty("clearColor", mtl.MTLClearColor{
-                    .red = @as(f32, @floatFromInt(self.current_background_color.r)) / 255,
-                    .green = @as(f32, @floatFromInt(self.current_background_color.g)) / 255,
-                    .blue = @as(f32, @floatFromInt(self.current_background_color.b)) / 255,
+                    .red = @as(f32, @floatFromInt(self.current_background_color.r)) / 255 * self.config.background_opacity,
+                    .green = @as(f32, @floatFromInt(self.current_background_color.g)) / 255 * self.config.background_opacity,
+                    .blue = @as(f32, @floatFromInt(self.current_background_color.b)) / 255 * self.config.background_opacity,
                     .alpha = self.config.background_opacity,
                 });
             }

--- a/src/renderer/OpenGL.zig
+++ b/src/renderer/OpenGL.zig
@@ -2344,9 +2344,9 @@ fn drawCellProgram(
 
     // Clear the surface
     gl.clearColor(
-        @as(f32, @floatFromInt(self.draw_background.r)) / 255,
-        @as(f32, @floatFromInt(self.draw_background.g)) / 255,
-        @as(f32, @floatFromInt(self.draw_background.b)) / 255,
+        @as(f32, @floatFromInt(self.draw_background.r)) / 255 * self.config.background_opacity,
+        @as(f32, @floatFromInt(self.draw_background.g)) / 255 * self.config.background_opacity,
+        @as(f32, @floatFromInt(self.draw_background.b)) / 255 * self.config.background_opacity,
         @floatCast(self.config.background_opacity),
     );
     gl.clear(gl.c.GL_COLOR_BUFFER_BIT);

--- a/src/renderer/OpenGL.zig
+++ b/src/renderer/OpenGL.zig
@@ -2344,9 +2344,9 @@ fn drawCellProgram(
 
     // Clear the surface
     gl.clearColor(
-        @as(f32, @floatFromInt(self.draw_background.r)) / 255 * self.config.background_opacity,
-        @as(f32, @floatFromInt(self.draw_background.g)) / 255 * self.config.background_opacity,
-        @as(f32, @floatFromInt(self.draw_background.b)) / 255 * self.config.background_opacity,
+        @floatCast(@as(f32, @floatFromInt(self.draw_background.r)) / 255 * self.config.background_opacity),
+        @floatCast(@as(f32, @floatFromInt(self.draw_background.g)) / 255 * self.config.background_opacity),
+        @floatCast(@as(f32, @floatFromInt(self.draw_background.b)) / 255 * self.config.background_opacity),
         @floatCast(self.config.background_opacity),
     );
     gl.clear(gl.c.GL_COLOR_BUFFER_BIT);

